mysql-test/suite/galera_3nodes/t/galera_sst_donor_non_prim.cnf

```diff
@@ -0,0 +1,4 @@
+!include ../galera_3nodes.cnf
+
+[mysqld.2]
+wsrep_sst_donor=node1
```

Comment:
seems like another typo

Reply:
It is `wsrep_sst_donor`, not `wsrep_sst_name`.